android/src/main/kotlin/com/rouninlabs/another_brother/AnotherBrotherPlugin.kt

```diff
@@ -125,6 +125,9 @@ class AnotherBrotherPlugin : FlutterPlugin, MethodCallHandler {
         }
         else if (call.method == ReplaceTextNameMethodCall.METHOD_NAME) {
             ReplaceTextNameMethodCall(flutterAssets = mFlutterAssets, context = mContext, call = call, result = result).execute()
+        }
+		else if (call.method == FlushPttPrintMethodCall.METHOD_NAME) {
```

Comment:
Thanks for the update @rendesg. Looks great, if you wouldn't mind could you fix up the formatting here and we'll go ahead and merge this to main. 

Thanks again!

Reply:
Sorry, I was using some plain text editor so I have accidentally mixed up the formatting and it looked fine for me (tabs vs spaces).
I hope it's fine now!